app/src/main/java/com/fstyle/androidtrainning/data/local/roomdb/dao/TrackDAO.java

```diff
@@ -0,0 +1,25 @@
+package com.fstyle.androidtrainning.data.local.roomdb.dao;
+
+import android.arch.persistence.room.Dao;
+import android.arch.persistence.room.Insert;
+import android.arch.persistence.room.Query;
+import com.fstyle.androidtrainning.data.local.roomdb.entity.TrackEntity;
+import com.fstyle.androidtrainning.model.Track;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.model.Track. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports